recipe/meta.yaml

```diff
@@ -9,7 +9,7 @@ source:
   sha256: 051fb31336170692c6a4cb47334315ec7fba8be92316e208183ab157739c0855
 
 build:
-  number: 0
+  number: 1000
```

Comment:
This one did not have the `host` section and was not ready for the migration :-/